### PR TITLE
WX-853 Remove CWL language factory, Centaur runner

### PR DESCRIPTION
**Pull request by aednichols**
This code should be inert following https://github.com/broadinstitute/cromwell/pull/6955 but seemed to cause at least one local development issue

### CODE REVIEW

**Comment by aednichols** [comment]
Fully refreshing local SBT was found to fix the dev issue, but this changeset would have made it less confusing.

File: project/ContinuousIntegration.scala [hunk 1]
```diff
@@ -67,9 +67,7 @@ object ContinuousIntegration {
   /**
     * For "reasons" these projects are excluded from the root aggregation in build.sbt.
     */
-  private val unaggregatedProjects = Map(
-    "cwlEncoder" -> "not sure what this is"
-  )
+  private val unaggregatedProjects = Map.empty
```
Comment (aednichols) [inline project/ContinuousIntegration.scala:70]: This change is not strictly related and more the result of auditing non-comment references to "cwl" in `.scala` files.

**Review by kraefrei — APPROVED** [review]
Appreciate this cleanup 👍

**Review by jgainerdewar — APPROVED** [review]
